File: C++/1985.cpp
#include <bits/stdc++.h>
 
using namespace std;
 
int main() {
 
    int a;
    float total = 0;
    cin >> a;
    for(int i = 0; i<a; i++){
        int cod, qnt;
        cin >> cod >> qnt;
        if(cod == 1001){ total += 1.50 * qnt;}
        else if(cod == 1002){ total += 2.50 * qnt;}
        else if(cod == 1003){ total += 3.50 * qnt;}
        else if(cod == 1004){ total += 4.50 * qnt;}
        else if(cod == 1005){ total += 5.50 * qnt;}
    }
    printf("%.2f\n", total);
}

File: Python/1985.py
total = 0
for i in range(int(input())):
    
    ped, qnt = map(int, input().split())
    
    if ped == 1001:
        total += qnt * 1.5
    elif ped == 1002:
        total += qnt * 2.5
    elif ped == 1003:
        total += qnt * 3.5
    elif ped == 1004:
        total += qnt * 4.5
    else:
        total += qnt * 5.5
        
print(f'{total:.2f}')